modules/integration/tests/src/test/java/org/wso2/appmanager/integration/utils/HttpUtil.java

```diff
@@ -33,79 +32,31 @@
     /**
      * Send a PUT request.
      * @param endpoint Url.
-     * @param postBody String.
+     * @param putBody String.
      * @param headers Map<String, String>.
      * @return httpResponse HttpResponse.
-     * @throws Exception on errors.
+     * @throws IOException on errors.
      */
-    public static HttpResponse doPut(URL endpoint, String postBody,
-                                     Map<String, String> headers) throws Exception {
+    public static HttpResponse doPut(URL endpoint, String putBody, Map<String, String> headers) throws IOException {
         HttpURLConnection urlConnection = null;
+        OutputStream out = null;
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/pull/357/commits/d5adfb28b2122712d0da0911c0d4921cf5bd98d9